.github/workflows/preview.yml

```
@@ -64,6 +64,14 @@ jobs:
         env:
           PLATFORM: linux/amd64
 
+      - name: chmod
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:20: Double quote to prevent globbing and word splitting

(shellcheck)